src/services/rocketChat/helpers.js

```diff
@@ -1,10 +1,9 @@
 const {
-	REQUEST_TIMEOUT,
 	ROCKET_CHAT_URI,
 	ROCKET_CHAT_ADMIN_TOKEN,
 	ROCKET_CHAT_ADMIN_ID,
 } = require('../../../config/globals');
-
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `../../../config/globals`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)